docs/manual/regex-checker.tex

```diff
@@ -50,7 +50,7 @@
   is an array that specifies the capturing groups that are definitely
   non-null (possibly empty) provided the regular expression matches the
   target string,. For example if an expression's type is
-  \<RegexNNGroups(groups = 2, nonNullGroups = {1})>, then it's run-time
+  \<RegexNNGroups(groups = 2, nonNullGroups = {1})>, then its run-time
   value could be \<"colo(u?)r(s)?"> but not \<"colo(u)?r(s?)"> nor a
   non-regex string such as \<"1) first point">. Note that \code{Regex(1)}
   is exactly the same as \code{EnhancedRegex(groups = 1)}.
```

Comment:
I don't think they are exactly the same.  The one capturing group might not match.  Can you give an example of this?